### PR TITLE
About reports link

### PR DESCRIPTION
**Pull request by ericliddle**

### CODE REVIEW

**Review by wtLau — CHANGES_REQUESTED** [review]
Changes to annual reports is good; but changes to footer and join-us is extra for this pull request

File: themes/sgsc/sass/_about.scss [hunk 1]
```diff
@@ -1,11 +1,19 @@
 .page-template-page-about {
+  .entry-content {
+    padding: 0;
+    margin: 0;
+    margin-top: 1rem;
```
Comment (kmsolberg) [inline themes/sgsc/sass/_about.scss:5]: Can you make this margin smaller?

**Review by kmsolberg — CHANGES_REQUESTED** [review]

File: themes/sgsc/sass/_about.scss (hunk 1 above)
Comment (ericliddle) [reply themes/sgsc/sass/_about.scss:5]: Unless it's changing this margin on the About page is equal to that of the Programs page.


About:
image

Programs:
image

Comment (kmsolberg) [reply themes/sgsc/sass/_about.scss:5]: image

Comment (kmsolberg) [reply themes/sgsc/sass/_about.scss:5]: The proportions of everything should be like programs

**Pull request merged**